public/app/features/org/OrgDetailsPage.tsx

```diff
@@ -32,14 +33,17 @@ export class OrgDetailsPage extends PureComponent<Props> {
   render() {
     const { navModel, organization } = this.props;
     const isLoading = Object.keys(organization).length === 0;
+    const canViewOrg = contextSrv.hasPermission(AccessControlAction.OrgsRead);
```

Comment:
nit: would it make sense use `canReadOrg` instead, for the sake of consistency? (the same for other bool's)

Reply:
ok!

Reply:
I agree.